src/Twig/AppExtension.php

```diff
@@ -11,15 +11,12 @@
 
 namespace App\Twig;
 
-use App\Utils\Markdown;
 use Symfony\Component\Intl\Locales;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFilter;
 use Twig\TwigFunction;
 
 /**
- * This Twig extension adds a new 'md2html' filter to easily transform Markdown
- * contents into HTML contents inside Twig templates.
  *
```

Comment:
This empty line could probably be removed.

Reply:
done